src/containers/target-pane.jsx

```diff
@@ -127,8 +132,9 @@ class TargetPane extends React.Component {
     }
     handleSpriteUpload (e) {
         const storage = this.props.vm.runtime.storage;
+        const costumeSuffix = this.props.intl.formatMessage(sharedMessages.costume, {index: 1});
         handleFileUpload(e.target, (buffer, fileType, fileName) => {
-            spriteUpload(buffer, fileType, fileName, storage, this.handleNewSprite);
+            spriteUpload(buffer, fileType, fileName, storage, this.handleNewSprite, costumeSuffix);
```

Comment:
These upload functions are super overloaded...  I'm pretty confused by `costumeSuffix` here — why does sprite upload need a suffix?

Reply:
Sprite upload creates a costume from the image uploaded. The name of the costume is the filename with `-costume1` appended. Rather than making fileupload locale aware, I added the costume suffix so the caller can take care of localizing. The caller is already where localizing new sprite happens.